File: TI1.3 REST_JAVA_CONUNI_GRP3/01 SERVIDOR/src/main/java/ec/edu/espe/conversionapi/enums/WeightUnit.java
package ec.edu.espe.conversionapi.enums;

/**
 * Unidades de peso
 */
public enum WeightUnit {
    KILOGRAMS("Kilogramos"),
    GRAMS("Gramos"),
    MILLIGRAMS("Miligramos"),
    POUNDS("Libras"),
    OUNCES("Onzas"),
    TONS("Toneladas");

    private final String displayName;

    WeightUnit(String displayName) {
        this.displayName = displayName;
    }

    public String getDisplayName() {
        return displayName;
    }
}

File: TI1.4 REST_DOTNET_CONUNI_GRP3/02 CLIMOV/convrtfull/app/src/main/java/ec/edu/espe/conv_rtfull/data/model/WeightUnit.kt
package ec.edu.espe.conv_rtfull.data.model

/**
 * Unidades de peso
 */
enum class WeightUnit(val displayName: String, val code: Int) {
    KILOGRAMS("Kilogramos", 0),
    GRAMS("Gramos", 1),
    MILLIGRAMS("Miligramos", 2),
    POUNDS("Libras", 3),
    OUNCES("Onzas", 4),
    TONS("Toneladas", 5)
}
